salt/orchestrate/aws/mitx_qa.sls

```diff
@@ -310,3 +303,34 @@ create_salt_master_security_group:
             - 10.0.0.0/16
     - require:
         - boto_vpc: create_mitx_qa_vpc
+
+create_vault_backend_security_group:
+  boto_secgroup.present:
+    - name: vault-{{ VPC_RESOURCE_SUFFIX }}
+    - vpc_name: {{ VPC_NAME }}
+    - description: >-
+        ACL to allow Vault to access data stores so that it
+        can create dynamic credentials
+    - tags:
+        Name: salt_master-{{ VPC_RESOURCE_SUFFIX }}
+    - rules:
+        {% MongoDB %}
+        - ip_protocol: tcp
+          from_port: 27017
+          to_port: 27017
+          cidr_ip:
+            - 10.0.0.0/16
```

Comment:
Should we restrict this more specifically to the 3 public subnets instead of the entire VPC?